### ISSUE TITLE
Rule proposal: display-name should be capitalized

### ISSUE THREAD

**Issue by Dashed** [yannickcr/eslint-plugin-react#170]
Good:
```js
React.createClass({ displayName:"Foo", ... });
```

Bad:
```js
React.createClass({ displayName:"foo", ... });
```

may be related to https://github.com/yannickcr/eslint-plugin-react/issues/157

**Comment by oliviertassinari** [comment]
:+1:
